react-native-2/src/main/scala/hello/world/Styles.scala

```diff
@@ -0,0 +1,31 @@
+package hello.world
+
+import scala.scalajs.js
+import scala.scalajs.js.Dynamic.literal
+
+object Styles {
+  val headerStyle = literal(
```

Comment:
Can we use `TextStyle` / `ViewStyle` instead of literals?

Reply:
Hmmm... I think they don't play well with `slinky.native.{Text, View}` but honestly I haven't examined the issue in depth